fw/rbcx-coprocessor/src/Dispatcher.cpp

```diff
@@ -32,6 +33,8 @@ void dispatcherPoll() {
             status.which_payload = CoprocStat_buttonsStat_tag;
             controlLinkTx(status);
             break;
+        case CoprocReq_setStupidServo_tag:
+            stupidServoDispatch(request.payload.setStupidServo);
```

Comment:
Chybí break;, zatím nevadí, ale aby se na to nezapomnělo.

Reply:
jéjé :D

Reply:
Přidávám výstupy pro všechny kanály, už je to spravený